packages/server/graphql/types/Reactji.ts

```diff
@@ -20,6 +27,13 @@ const Reactji = new GraphQLObjectType<ReactjiDB, GQLContext>({
       type: new GraphQLNonNull(GraphQLInt),
       description: 'The number of users who have added this reactji'
     },
+    users: {
+      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(require('./User').default))),
+      description: 'The users who added a reactji',
+      resolve: (parent, _args: any, {dataLoader}) => {
```

Comment:
+1 could destructure `parent` to `{userId`} and add the type, then change `_args` from `any` to `unknown`